gef.py

```diff
@@ -7733,18 +7733,26 @@ def reason():
             err("No thread selected")
             return
 
+        selected_thread = gdb.selected_thread()
+
         for i, thread in enumerate(threads):
-            line = """[{:s}] Id {:d}, Name: "{:s}", """.format(Color.colorify("#{:d}".format(i), "bold pink"),
-                                                               thread.num, thread.name or "")
+            line = """[{:s}] Id {:d}, """.format(Color.colorify("#{:d}".format(i), "bold green" if thread==selected_thread  else "bold pink"), thread.num)
+            if thread.name:
+                line += """Name: "{:s}", """.format(thread.name)
             if thread.is_running():
                 line += Color.colorify("running", "bold green")
             elif thread.is_stopped():
                 line += Color.colorify("stopped", "bold red")
+                thread.switch()
+                frame = gdb.selected_frame()
+                line += " {:s} in {:s} ()".format(Color.colorify(hex(frame.pc()), "blue"),Color.colorify(frame.name() or "??" ,"bold yellow"))
```

Comment:
Please use a format string instead of `hex` on the pc

Reply:
```suggestion
                line += " {:s} in {:s} ()".format(Color.colorify("{:#x}".format(frame.pc()), "blue"),Color.colorify(frame.name() or "??" ,"bold yellow"))
```
will it be OK?

Reply:
Yup that's good.